.gitignore

```diff
@@ -1,3 +1,4 @@
 # Ignore bin and obj files
 bin/
 obj/
+.vscode/
```

Comment:
add empty line in this file, also, ignore obj and bin files, there are tow bin and three obj in this PR